tests/printer/printer.cpp

```diff
@@ -308,13 +309,14 @@ TEST(Printer, printModelImportingModelParentComponent)
     auto printer = libcellml::Printer::create();
     auto serialisedModel = printer->printModel(model);
 
-    EXPECT_EQ(serialisedModel, fileContents("importingModelParentComponent.cellml"));
+    EXPECT_EQ(serialisedModel, modelContents);
```

Comment:
These entries should be the other way around, we expect 'modelContents' to be equal to what we actually got 'serialiseModel' after our manipulations.

Reply:
Good point. Done in commit 2af7803c.